ansible/roles/ansible-horizon-plugin/tasks/remove_pip_packages.yml

```diff
@@ -0,0 +1,15 @@
+- name: copy remove_pip_packages.sh file to tmp dir
+  template:
+    src: remove_pip_packages.sh
+    dest: /tmp/remove_pip_packages.sh
+
+- name: Assign Execute permission /tmp/remove_pip_packages.sh
+  shell: chmod +x /tmp/remove_pip_packages.sh
+
+- name: Run the script /tmp/remove_pip_packages.sh
+  shell: ./remove_pip_packages.sh
+  args:
+    chdir: /tmp
+
+- name: Remove /tmp/remove_pip_packages.sh
+  file: path="/tmp/remove_pip_packages.sh" state=absent
```

Comment:
we won't require this. we dont use pip packages anymore

Reply:
code removed